tests/packages/sharedlib-in-package/mypkg/__init__.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: 2024 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import sys
+
+
+# Note: the below function is used in the docs via a literalinclude directive.
+#       When making any changes to this file, please ensure the right set of
+#       lines is pulled into the docs, and adjust the :lines: attribute of
+#       the literalinclude if needed.
+
+
+def _enable_sharedlib_loading():
+    """
+    Ensure the shared libraries in this dir and the `sub` subdir can be loaded
+    on Windows.
+
+    One shared library is installed alongside this ``__init__.py`` file.
```

Comment:
There is an inconsistent use of single and double backticks used to delimit literals in this docstring. It does not matter much, but as it is a code example maybe we should be a bit more careful than usual.

Reply:
Good point. I did them more carefully, only keeping the stdlib module/object that will get linked almost certainly correctly by intersphinx as single backtick (link), and the rest as double backtick (code). That way it works well when copying into some other project.